src/liballoc/string.rs

```diff
@@ -384,6 +384,38 @@ impl String {
         String { vec: Vec::new() }
     }
 
+    #[inline]
+    #[unstable(feature = "lit_strings", reason = "its TOO lit", issue = "42069")]
```

Comment:
You might want to change this to something more informative.

Reply:
This isn't intended to be landed any time soon, so I opted for a joke :)

Reply:
> This isn't intended to be landed any time soon

Do you say this because there is more implementation work to be done, or because it hasn't been decided that this is a good thing? (I'm pretty out of the loop.) If the former, cool. If the latter, has it been discussed somewhere?

Reply:
There's a thread on internals about landing this feature as a coercion (rather than a method): https://internals.rust-lang.org/t/pre-rfc-allowing-string-literals-to-be-either-static-str-or-string-similar-to-numeric-literals/5029

Reply:
Oh hey, that's pretty cool! Thanks.